pages/common/tmsu.md

```diff
@@ -0,0 +1,28 @@
+# tmsu
+
+> Simple command-line tool for tagging files.
+> More information: <https://tmsu.org>.
+
+- Tag a file:
```

Comment:
```suggestion
- Tag a specific file with multiple tags:
```

Reply:
@vlfldr I think if we want to accept the suggestion(s), then we would probably want to `Commit Suggestion` instead of `Resolve Conversation` :)